prep-example/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>prep-example</title>
+  </head>
+  <body>
+    <h1>I would like to build</h1>
+    <p>
+      <!-- write your app ideas here 👇 -->
+    </p>
```

Comment:
You should have some app ideas that you write down inside of here

Reply:
ok let me figure out how to do that